data/js/emoji.js

```diff
@@ -1,5 +1,5 @@
-/*global twemoji, self*/
-/*jslint browser: true, devel: true*/
+-/*global twemoji*/
+-/*jslint browser: true, devel: true*/
 if (twemoji === undefined) {
     console.error("Can't load Twemoji");
 } else {
```

Comment:
`self` does not exist anymore in webextensions. But we can do something like this:
```js
twemoji.base = browser.extension.getURL('data/components/twemoji/2/');
```